database/migrations/2020_02_16_151556_create_senses_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSensesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('senses', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('name');
+            $table->string('code');
```

Comment:
senses also have a range in feet

Some characters have darkvision 30ft, others 50ft, 60ft etc.

Reply:
never mind, just saw thats in the pivot